python/phylanx/ast/oscop.py

```diff
@@ -40,6 +39,11 @@ def __init__(self, root, kwargs):
         self.__src__ = self.template.substitute(
             self.oscop_global) + '\n' + self.__src__
 
+        if kwargs["dump_openscop"] == True:
```

Comment:
It might be useful to allow for the user to specify a file name where to dump the data to. the named argument could be used for that.